<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLabDlg</class>
 <widget class="QDialog" name="QtLabDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Laboratory</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="rectBtn">
       <property name="text">
        <string>rectangle</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="roundRectBtn">
       <property name="text">
        <string>round rect</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="circleBtn">
       <property name="text">
        <string>circle</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="polygonBtn">
       <property name="text">
        <string>polygon</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mergeBtn">
       <property name="text">
        <string>merge</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="overlapsBtn">
       <property name="text">
        <string>overlaps</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>clear</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="layerProfileBtn">
       <property name="text">
        <string>layer profile</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="mainWgt" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
